tests/release/release.go

```diff
@@ -0,0 +1,136 @@
+package release
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/release-service/api/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"knative.dev/pkg/apis"
+)
+
+const (
+	snapshotName          = "snapshot"
+	sourceReleaseLinkName = "source-release-link"
+	targetReleaseLinkName = "target-release-link"
+	releaseStrategyName   = "strategy"
+	releaseName           = "release"
+	releasePipelineName   = "release-pipeline"
+	applicationName       = "application"
+	releasePipelineBundle = "quay.io/hacbs-release/demo:m5-alpine"
+
+	avgPipelineCompletionTime = 2 * time.Minute
+	defaultInterval           = 100 * time.Millisecond
+)
+
+var snapshotImages = []v1alpha1.Image{
```

Comment:
Similar to previous comments, `Images` changes to `Components` soon

Reply:
as above